Sources/Operators.swift

```diff
@@ -0,0 +1,236 @@
+//
```

Comment:
We should take a dependency on Operadics, not include it directly.

Reply:
Good spot. Not sure how that got in there.